src/modules/blockchainApplication/explore/manager/BlockchainApplicationStatistics.js

```diff
@@ -1,8 +1,8 @@
 /* istanbul ignore file */
 import { compose } from 'redux';
 import withData from 'src/utils/withData';
-import ManageBlockchainApplicationsView from '../../components/ManageBlockchainApplications';
-// import { getApps, getStatistics } from '../../../explore/utils/api';
+import ManageBlockchainApplicationsView from '../components/BlockchainApplications';
+// import { getStatistics } from '../../../explore/api';
```

Comment:
This file seems outdated from `feature/blockchain-application`. Please check `src/modules/blockchainApplucation/manage/manager/ManageBlockchainApplications/BlockchainApplicationsStatistics.js` and move it to here if need. `getStatistics` functions was unmocked and uncomented

Reply:
@isalga so API directory is on the root of each domain so I just fix the comment path and I do blive its correct the correct path of not please mention where and why

Reply:
Ok, my bad, it was unmocked and uncomented in another ticket in progress (https://github.com/LiskHQ/lisk-desktop/pull/4382) and not in feature branch

Reply:
then please after merge this branch correct the other one